assets/js/main.js

```diff
@@ -1,15 +1,14 @@
 async function getMovies() {
-  const result = await fetch("http://localhost:8080/");
+  const result = await fetch("https://api.themoviedb.org/3/movie/top_rated?api_key=9f120f0e95e5cedf05a78abda9ef8477&language=en-US&page=1");
```

Comment:
you already have `apiKey` in a variable.

Reply:
Done.